lib/index.js

```diff
@@ -22,15 +22,44 @@ var convert
     , reactiveEnergy: require('./definitions/reactiveEnergy')
     , volumeFlowRate: require('./definitions/volumeFlowRate')
     }
-  , Converter;
+  , Converter
+  , definitions
+  , InvalidDefinitionError = require('./invalidDefinitionError');
 
 Converter = function (numerator, denominator) {
-  if(denominator)
+  if (!definitions) {
+    this.resetDefinitions();
+  }
+
+  if (denominator)
     this.val = numerator / denominator;
   else
     this.val = numerator;
 };
 
+Converter.prototype.resetDefinitions = function () {
+  definitions = {};
+  this.addDefinitions(initialDefinitions);
+
+  return this;
+};
+
+Converter.prototype.addDefinitions = function (newDefinitions) {
+  each(newDefinitions, function(definition, type) {
+    if (!isValidDefinition(definition)) {
+      throw new InvalidDefinitionError(type);
+    }
+
+    definitions[type] = definition;
+  });
+
+  return this;
+};
```

Comment:
I'm not a fan of `resetDefinitions` and `addDefinitions` mutating the `definitions` variable in the module's scope.

Each instance of the converter (created with `convert()`) should get its own `definitions`. Otherwise, two modules that both depend on `convert-units` could change each others' behavior, and be very difficult to debug. It's basically like having a global variable.